odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -197,3 +197,23 @@ Stopping ODK-X Sync Endpoint
 
       $ docker volume rm (docker volume ls -f "label=com.docker.stack.namespace=syncldap" -q)
 
+.. _sync-anonymous:
```

Comment:
Could you add links to this section in https://docs.odk-x.org/sync-endpoint-cloud-setup?